custom/ewsghana/templates/ewsghana/partials/ews_multibar_chart.html

```diff
@@ -55,8 +55,19 @@
         multibar_chart.multibar.groupSpacing(chart_config.groupSpacing);
         multibar_chart.multibar.forceY(chart_config.forceY);
 
+        var data =  JSON.parse(JSON.stringify(multibar_chart_data));
+        var productCodes = [];
+        $("#report_table_product_selection_pane input:not(:checked)").each(function () {
+            productCodes.push($(this).attr('value'));
+        });
+        for(var i = 0; i < data.length; i++) {
+            data[i].values = jQuery.grep(data[i].values, function(value) {
+                return jQuery.inArray(value.x, productCodes) === -1;
```

Comment:
underscore has a nice utility for this so you don't have to check -1. 
```
return _.contains(productCodes, value.x)
```
and that returns a boolean